docs/spfx/library-component-tutorial.md

```diff
@@ -55,15 +55,24 @@ localization_priority: Priority
 1. Run `gulp` on the command prompt to see everything builds fine.
 
 ## How to consume a 3rd party SPFx library (for local testing)
-1. Run `npm link` from the root directory of library solution. In this case it would be from the **```corporate-library```** folder.
+1. Create a web part project in a **separate project folder, so not in the library project folder structure**, following the instructions from [here](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/web-parts/get-started/build-a-hello-world-web-part#to-create-a-new-web-part-project). Name your web part 'CorporateWebPart'. Like the library before the web part also need to target to ***SharePoint Online only (latest)**.
```

Comment:
Don't use "here" as the link, include the name of the document (accessibility reasons)

Reply:
Taken unmodified from the original documentation but you are right.

Reply:
Also... "CorporateWebPart" shouldn't be code formatted. Should be bold.